app/src/features/Batches/BatchCreate.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from "react";
+
+import {
+  Button,
+  Chip,
+  FormControl,
+  Input,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+  useTheme,
+} from "@material-ui/core";
+import { makeStyles, Theme } from "@material-ui/core/styles";
+import PlayIcon from "@material-ui/icons/PlayCircleOutline";
+import { useTranslation } from "react-i18next";
+import { useParams } from "react-router-dom";
+
+import {
+  useApiResourcesListQuery,
+  useApiBatchesCreateMutation,
+} from "services/api/endpoints";
+import type { Resource } from "services/api/generated/api.generated";
+
+import Alert from "../../common/components/Alert";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+    maxWidth: 300,
+  },
+  button: {
+    margin: theme.spacing(1),
+    marginTop: "auto",
+  },
+  chips: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  chip: {
+    margin: 2,
```

Comment:
I'd prefer if we used `theme.spacing()` for defining sizes, margins, paddings, etc..